src/test/resources/logback.xml

```diff
@@ -9,12 +9,12 @@
     </appender>
 <!--    uncomment and set to DEBUG to log all failing HTTP requests-->
 <!--    uncomment and set to TRACE to log all HTTP requests-->
-    <logger name="io.gatling.http.engine.response" level="TRACE" />
+    <logger name="io.gatling.http.engine.response" level="INFO" />
     <!-- Sets the performance-test-runner log level-->
     <logger name="uk.gov.hmrc.performance" level="INFO"/>
```

Comment:
maybe comment out line 24 as well

Reply:
Hi Thad, I am confused now. There is no line 24 in logback.xml .